vizalerts.py

```diff
@@ -1045,29 +1044,41 @@ def get_password_from_file(password):
         sys.exit(1)
 
 
-def find_vizimages_in_body(data, has_email_header, has_email_footer):
-    """Returns list of all found occurences of VIZ_IMAGE() in email body text"""
+
+def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
+    # updated 20160223 Jonathan Drummey
```

Comment:
Minor quibble, but would you mind changing the comment block to this:

def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
"""
Return a list of all viewnames to be downloaded for the
body, header, and/or footer. The list is a distinct list
to prevent unnecessary downloading.
    updated 20160223 Jonathan Drummey
    function call now includes viewurlsuffix, return is now a list of viewnames
 """

That way the docstring can be used in documentation (should we ever need it)